code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -1878,6 +1888,11 @@
 	glass_center_of_mass = list("x"=16, "y"=2)
 	taste_tag = list(TASTE_SPICY,TASTE_DRY)
 
+/datum/reagent/alcohol/demonsblood/affect_ingest(mob/living/carbon/M, alien, effect_multiplier)
+	. = ..()
+	if(M.bodytemperature > 350)
```

Comment:
77°C body temperature? Lethal temperature is 44°C.
38-39°C is severe discomfort, could use that instead.
```suggestion
	if(M.bodytemperature > 311)
```

Reply:
Trigger temperature is now 311, though I would like to point out most of the warming drinks will be able to trigger this

Reply:
Current body temp handling is super ancient and poorly done, increasing temperature to crazy numbers sometimes, and in general making no sense.
If anything, we should improve that system, not adjust other features to it's nonsensical behavior.